roles/openshift_logging/templates/es.j2

```diff
@@ -37,7 +37,10 @@ spec:
           imagePullPolicy: Always
           resources:
             limits:
-              memory: "{{openshift_logging_es_instance_ram}}i"
+              memory: "{{es_memory_limit}}"
```

Comment:
We default this to "1024M" but for the resource limit it should be "1024Mi" (see Fluentd)